tests/console/zypper_info.pm

```diff
@@ -46,15 +46,15 @@ sub test_package_output {
 sub run {
     select_console 'root-console';
 
-    prepare_source_repo;
-
     if (is_leap('15.0+') and get_var('ARCH') =~ /aarch64|ppc64le/) {
         record_soft_failure(
             'for Leap:15.0:Ports aarch64 and ppc64le, do not enable source repo, waiting for solved issue https://progress.opensuse.org/issues/36256');
         zypper_call("ref");
```

Comment:
why do we call `zypper ref` when we did not add any repos then?

Reply:
We already have other repos, so let's refresh them.